tools/decdk/package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "build": "node ./deps.js && tsc && chmod +x bin/decdk && chmod +x bin/decdk-schema && bin/decdk-schema > cdk.schema.json",
     "watch": "tsc -w",
-    "test": "jest"
+    "test": "jest",
+    "package": "mkdir -p dist/js && cd dist/js && npm pack ../../"
```

Comment:
Woahhh... do we always have such a complex package command?

Reply:
Nope, most stuff uses `cdk-build`, but here I'm going for a "poor man's fix-this-now" so it publishes without too much fuss.  We can replace afterwards...